ui/app/templates/components/mount-backend-form.hbs

```diff
@@ -47,7 +47,7 @@
               @onRadioChange={{queue (action (mut this.mountModel.type)) (action "onTypeChange" "type")}}
               @disabled={{if type.requiredFeature (not (has-feature type.requiredFeature)) false}}
               @tooltipMessage={{if
-                (or (eq type.type "transform") (eq type.type "kmip"))
+                (or (eq type.type "transform") (eq type.type "kmip") (eq type.type "keymgmt"))
                 (concat
                   type.displayName
                   " is part of the Advanced Data Protection module, which is not included in your enterprise license."
```

Comment:
We should double-check that this is true -- not sure which module KMSE is actually in

Reply:
I'm not sure where to find that out. I can add a TODO comment so we can track that down before we finish phase 2.